addOns/revisit/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this add-on will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## 5 - 2023-10-15
+### Changed
```

Comment:
As mentioned in https://github.com/zaproxy/zap-extensions/blob/main/CONTRIBUTING.md#changelog this should be added to the Unreleased section. Should be under a `Fixed`.